README.md

```diff
@@ -47,8 +47,8 @@ spec:
 *Corresponding Operator Config*
 This configuration is the auto-generated translation of the above Cluster configuration.
 ```yaml
-apiVersion: networkoperator.openshift.io/v1
-kind: NetworkConfig
+apiVersion: operator.openshift.io/v1
```

Comment:
Could you provide the details for the example you mention above of changing the VXLANPORT. A specific question, do the repeated fields between the cluster config and operator config have to be stated and the same? How is this input to the installer?